rest/route/project.go

```diff
@@ -1120,5 +1121,10 @@ func (h *projectVarsPutHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.NewJSONInternalErrorResponse(errors.Wrapf(err,
 			"error updating projects with matching keys"))
 	}
+	_, err = artifact.RotateSecrets(h.replaceVars.ToReplace, h.replaceVars.Replacement, h.replaceVars.DryRun)
```

Comment:
should i be trying to return this as well? not sure exactly how many artifact files well be changing

Reply:
Yeah this is a good point, it might take long enough that this would time out, and I'm not sure there's a way to test if that's true. Can we add a flag that's like "rotate_files": true or something to the body? I would suggest in the docs that people first rotate the secrets, and _then_ rotate the files, since that's less critical. (Or alternatively rotating the secret for the files should be it's own route, but it's probably not worth that work)

Reply:
added rotate files and will update doc later!